RocketChat/Rocket.Chat
# [FIX] Showing Workspace Section For User Having LiveChat Manager Permission

## Proposed changes (including videos or screenshots)





## Issue(s)
Closes #27081


## Steps to test or reproduce
Please refer #27081 for more details


1. Log in as `livechat-manager` user (should not have admin role)
2. Open `Administration > Workspace` menu item
![Screenshot 2022-12-27 at 11 12 35](https://user-images.githubusercontent.com/9275105/209679323-11a320f6-914e-4c24-8b98-68bdc3d38d90.png)

## Further comments


The `Workspace` menu shouldn't be visible since the `livechat-manager` user doesn't have any admin permission.
